package.json

```diff
@@ -93,8 +93,5 @@
     "events": "^1.1.1",
     "lodash": "^4.17.5",
     "qs": "^6.5.1"
-  },
-  "peerDependencies": {
```

Comment:
optionalPeerDependencies would fit best here I think

Reply:
@Haroenv Sorry, missed your comment - why is it needed as a peer dependency? it doesn't seem used in the `src` directory 🤔

Reply:
Because the library doesn't make sense with algoliasearch v2 or v4, so we need some way to explicitly say only v3 is correct?